sql/moz-fx-data-shared-prod/telemetry/clients_daily_v6/view.sql

```diff
@@ -13,7 +13,161 @@ SELECT
+      scalar_parent_urlbar_picked_autofill_origin_sum
+    ) AS scalar_parent_urlbar_picked_autofill_origin_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_autofill_other_sum
+    ) AS scalar_parent_urlbar_picked_autofill_other_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_autofill_preloaded_sum
+    ) AS scalar_parent_urlbar_picked_autofill_preloaded_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_autofill_url_sum
+    ) AS scalar_parent_urlbar_picked_autofill_url_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_bookmark_sum
+    ) AS scalar_parent_urlbar_picked_bookmark_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_dynamic_sum
+    ) AS scalar_parent_urlbar_picked_dynamic_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_extension_sum
+    ) AS scalar_parent_urlbar_picked_extension_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_formhistory_sum
+    ) AS scalar_parent_urlbar_picked_formhistory_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_history_sum
+    ) AS scalar_parent_urlbar_picked_history_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_keyword_sum
+    ) AS scalar_parent_urlbar_picked_keyword_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_remotetab_sum
+    ) AS scalar_parent_urlbar_picked_remotetab_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_searchengine_sum
+    ) AS scalar_parent_urlbar_picked_searchengine_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_searchsuggestion_sum
+    ) AS scalar_parent_urlbar_picked_searchsuggestion_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_switchtab_sum
+    ) AS scalar_parent_urlbar_picked_switchtab_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_tabtosearch_sum
+    ) AS scalar_parent_urlbar_picked_tabtosearch_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_tip_sum
+    ) AS scalar_parent_urlbar_picked_tip_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_topsite_sum
+    ) AS scalar_parent_urlbar_picked_topsite_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_unknown_sum
+    ) AS scalar_parent_urlbar_picked_unknown_sum,
+    `mozfun.map.extract_keyed_scalar_sum`(
+      scalar_parent_urlbar_picked_visiturl_sum
+    ) AS scalar_parent_urlbar_picked_visiturl_sum,
+    -- `mozfun.map.extract_keyed_scalar_sum`(
+    --   contextual_services_quicksuggest_block_dynamic_wikipedia_sum
+    -- ) AS contextual_services_quicksuggest_block_dynamic_wikipedia_sum,
```

Comment:
The commented out fields are the fields that don't exist yet. Since these are new, I'm thinking we can do the extract in the table and then delete these commented out lines?